### PR TITLE
[DBAAS-1239] Update Atlas operator to use OpenShift Database Access  v1beta1 API

### PR DESCRIPTION
**Pull request by jianrongzhang89**
### All Submissions:

* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Operator support Third Party Integration.
 
 ### Step 4. Test Database as a Service (DBaaS) on OpenShift
 
-The Atlas Operator is integrated with the [Red Hat Database-as-a-Service (DBaaS) Operator](https://github.com/RHEcosystemAppEng/dbaas-operator) which allows application developers to import database instances and connect to the databases through the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator). More information can be found [here](https://github.com/RHEcosystemAppEng/dbaas-operator#readme).
+The Atlas Operator is integrated with the [OpenShift Database Access Operator, a.k.a. Database-as-a-Service (DBaaS) Operator](https://github.com/RHEcosystemAppEng/dbaas-operator) which allows application developers to import database instances and connect to the databases through the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator). More information can be found [here](https://github.com/RHEcosystemAppEng/dbaas-operator#readme).
 
 Note that both the DBaaS Operator and Atlas Operator should be installed through the [Operator Lifecyle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager).
```
Comment (tchughesiv) [inline README.md]: Honestly, it probably wouldn't matter how Mongo Atlas was installed going forward... manually or via OLM. right?

Comment (jianrongzhang89) [reply README.md]: Right. I removed it.

**Review by tchughesiv — CHANGES_REQUESTED** [review]
looks like you have some `linter` issues to clear up. otherwise lgtm

**Review by tchughesiv — APPROVED** [review]
lgtm

**Pull request merged**